lib/fastlane/plugin/ddg_apple_automation/actions/tag_release_action.rb

```diff
@@ -99,10 +99,11 @@ def self.create_tag_and_github_release(is_prerelease, platform, github_token)
       end
 
       def self.merge_or_delete_branch(params)
-        branch = other_action.git_branch
         if params[:is_prerelease]
-          Helper::GitHelper.merge_branch(@constants[:repo_name], branch, params[:base_branch], params[:github_elevated_permissions_token] || params[:github_token])
+          # we actually merge the tag, not the branch
+          Helper::GitHelper.merge_branch(@constants[:repo_name], params[:tag], params[:base_branch], params[:github_elevated_permissions_token] || params[:github_token])
```

Comment:
This function uses Octokit under the hood. I verified in irb on a test branch and test tag that Octokit’s `client.merge` accepts tag name as a commit-ish being merged.